Source/SelfService/Web/microservice/microserviceView/configuration/setupSection.tsx

```diff
@@ -0,0 +1,268 @@
+                            port: 80,
+                            entrypoint: '',
+                            ingressPath: ingress.path.replace(/\//, '') || ''
+                        }}
+                        sx={styles.form}
+                    >
+                        <Box sx={styles.formSections}>
+                            <Typography variant='subtitle1' sx={{ mb: 2 }}>Configuration Setup</Typography>
+
+                            <Input id='microserviceName' label='Microservice Name' required disabled />
+                            <Input id='developmentEnvironment' label='Development Environment' disabled />
+
+                            <Select
+                                id='runtimeVersion'
+                                label='Runtime Version*'
+                                options={runtimeVersions}
+                                value={selectedRuntimeImage}
+                                required
+                                disabled={formIsNotEditable}
+                                onChange={(event) => setSelectedRuntimeImage(event.target.value)}
+                                sx={{ width: 220 }}
+                            />
+                        </Box>
+
+                        <Box sx={styles.formSections}>
+                            <Typography variant='subtitle2' sx={{ mb: 2 }}>Container Image Settings</Typography>
+
+                            <Input id='imageName' label='Image Name' required disabled={formIsNotEditable} sx={{ width: 500 }} />
+                            <Input id='port' label='Port' required disabled={formIsNotEditable} />
+                            <Input id='entrypoint' label='Entrypoint' disabled={formIsNotEditable} />
+
+                            <HeadArguments cmdArgs={headCommandArgs} setCmdArgs={setHeadCommandArgs} disabled={formIsNotEditable} />
+                        </Box>
+
+                        <Box sx={styles.formSections}>
+                            <Typography variant='subtitle2'>Public Microservice</Typography>
+
+                            <SwitchToggle
+                                title='Expose to a public URL'
+                                disabled={formIsNotEditable}
+                                checked={hasPublicURL}
+                                onChange={(event) => setHasPublicURL(event.target.checked)}
+                                sx={{ my: 2.5 }}
+                            />
+
+                            {hasPublicURL &&
+                                <Input
+                                    id="ingressPath"
+                                    label='Path'
+                                    startAdornment='/'
+                                    placeholder='leave blank for default path'
+                                    sx={{ width: 226 }}
+                                />
+                            }
+                        </Box>
+
+                        {hasM3Connector &&
```

Comment:
This should use the derived variable `canUseM3Connector` as mentioned in the commend further up